src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1671,6 +1671,27 @@ else if (!fhirImmunization.getNotGiven()) {
 
 	}
 
+	@Override
+	public Condition tIndication2ConditionEncounter(Indication cdaIndication) {
+		Condition cond = tIndication2Condition(cdaIndication);
+		Coding conditionCategory = new Coding().setSystem("http://hl7.org/fhir/condition-category");
+		conditionCategory.setCode("encounter-diagnosis");
+		conditionCategory.setDisplay("Encounter Diagnosis");
+		cond.addCategory().addCoding(conditionCategory);
+		return cond;
+	}
+
+	@Override
+	public Condition tIndication2ConditionProblemListItem(Indication cdaIndication) {
+		Condition cond = tIndication2Condition(cdaIndication);
+		Coding conditionCategory = new Coding().setSystem("http://hl7.org/fhir/condition-category");
```

Comment:
Can you add the OID for this code to the tOid2Url function, and call that get the code system?

Reply:
I updated the indication2condition to either be 2 functions: indication2conditionProblemListItem and indication2conditionEncounter... so I can't really map the one indication OID to two different URLs...